common.css

```diff
@@ -84,11 +87,75 @@ span{
 }
 .page-container{
     display: grid;
-    grid-template-columns: 300px auto;
+    grid-template-columns: 18.75rem auto;
+}
+.quiz-heading-container{
+    display: flex;
+    justify-content: space-between;
+    margin: 2rem 2rem;
+}
+.quiz-btn-container{
+    display: flex;
+    flex-wrap: wrap;
+    margin: 2rem 2rem;
+}
+.quiz-btn-container .prev-button{
+    margin: 1rem 1rem 1rem 0rem;
+}
+.quiz-btn-container .next-button{
+    margin: 1rem 0rem ;
+}
+.quiz-heading-container .image-container{
+    display: none;
+}
+.quiz-info-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 75rem;
+}
+.quiz-info-container .image-container{
+    margin: 0rem 2rem;
+    max-height: 300px;
+    max-width: 400px;
```

Comment:
use rems if possible.

Reply:
Ya but i dont wanna change its width on the basis of font size